CIP-1694/README.fr.md

```diff
@@ -0,0 +1,1185 @@
+---
```

Comment:
```suggestion
---
Source: https://github.com/cardano-foundation/CIPs/blob/master/CIP-1694/README.md
```
We agreed via https://github.com/cardano-foundation/CIPs/pull/542#pullrequestreview-1513216962 to add the URL source so that readers can follow back to the original document, especially for important fields like `Authors` and `Discussions` (and to know to what file the commit hash in `Revision` is referring to).